client/js/components/map/map/DpOlMapLayer.vue

```diff
@@ -87,12 +74,27 @@ export default {
   },
 
   methods: {
+    /*
+     * In several cases is shown map.attribution.default: in the view in basic settings for the procedure,
+     * in the map by defining startMapSegment and
+     * in the map in the general view in Settings for configuration for map and layer
+     */
+    defaultAttributions () {
+      const currentYear = formatDate(new Date(), 'YYYY')
+      return this.attributions
+        ? this.attributions.replaceAll('{currentYear}', currentYear)
+        : Translator.trans('map.attribution.default', {
+          linkImprint: Routing.generate('DemosPlan_misccontent_static_imprint'),
+          currentYear
+        })
+    },
```

Comment:
I d# not understand in which way this change alters the behavior. It seems like the code is reformatted to use a ternary, but the logic remains the same. Moving from a computed property to a method should not change, either - so i don't get it.

I would expect the change in the places where default attribution is set. In DpOlMap, the `defaultAttribution` option is passed to DpOlMapLayer as `attributions`. So i would remove this option whereever the default attribution should show up. Admittedly, naming something `defaultAttribution` which, if ommitted, leads to showing the default attribution, is quite confusing but hey, naming is hard :)